src/HealthChecks.UI/Core/Notifications/WebHookFailureNotifier.cs

```diff
@@ -108,15 +109,23 @@ private async Task SendRequest(string uri, string name, string payloadContent)
                 _logger.LogError($"The failure notification for {name} has not executed successfully.", exception);
             }
         }
+
         private string GetFailedMessageFromContent(UIHealthReport healthReport)
         {
             var failedChecks = healthReport.Entries.Values
-                .Where(c => c.Status != UIHealthStatus.Healthy)
-                .Count();
+                .Count(c => c.Status != UIHealthStatus.Healthy);
 
             return $"There is at least {failedChecks} HealthChecks failing.";
         }
 
+        private string GetFailedDescriptionsFromContent(UIHealthReport healthReport)
+        {
+            var failedChecks = healthReport.Entries.Values

```

Comment:
Probably with Aggregate you can do select and string join in the same sentence

Reply:
Line 118 should be "There are"

Reply:
I have never used Aggregate. I hope I did it right.